### ISSUE TITLE
Even if lecture is available in 1080p the udeler still downloads in 720p

### ISSUE THREAD

**Issue by dataisys88** [FaisalUmair/udemy-downloader-gui#494]
**Sample lecture URL :** https://www.udemy.com/course/javascript-the-complete-guide-2020-beginner-advanced/learn/lecture/15918598#overview

**Issue :** The above URL content is available in 1080p, but when i select video quality as Highest or Auto in settings and start download.The result download content is always in 720p only._I am not able to download  1080p lecture in Udeler 1.8 version,it gets downloaded in 720p only._

**Note :** The above mentioned Sample lecture URL is just for testing , but the above mentioned Issue persist for all available 1080p lectures.

**Comment by Megagoth1702** [comment]
Yup, would love to see this fixed. I donated a few bucks to maybe motivate to fix this. 👍 

Thank you! :)
